pkgs/flox-cli-tests/default.nix

```diff
@@ -152,7 +152,7 @@ writeShellScriptBin PROJECT_NAME ''
   }
   ${
     if WATCHDOG_BIN == null then
-      "export WATCHDOG_BIN='flox-watchdog';"
```

Comment:
might be worth addng a comment why _this one_ requires `which` (for those not eager to dive into git blames :)

Reply:
Added